index.html

```diff
@@ -5,8 +5,153 @@
     <title>HTML Academy: Седона</title>
   </head>
   <body>
+  <header class="page-header">
+    <nav class="navigation">
+      <a href="http://htmlacademy.ru"><img class="logo" src="/images/logo-sedona-header.svg" width="140" height="70" alt="Логотип Город Седон."></a>
```

Comment:
```suggestion
      <a href="http://htmlacademy.ru"><img class="logo" src="images/logo-sedona-header.svg" width="140" height="70" alt="Логотип Город Седон."></a>
```
Так как это относительный путь. В данном случае от корня сайта. Поэтому путь не должен начинаться с символа `/`, иначе браузер не найдёт файл изображения. В этом можно убедиться, если посмотреть [опубликованную версию](https://htmlacademy-htmlcss.github.io/2181423-sedona-35/2/).

Здесь можно прочитать про [абсолютные и относительные пути](https://up.htmlacademy.ru/htmlcss/35/module/3/item/9).